agoric/contract/src/kreadKit.js

```diff
@@ -1055,15 +1079,21 @@ export const prepareKreadKit = async (
             const platformFee = multiplyBy(want.Price, platformFeeRate);
 
             const object = objectInSellSeat.value.payload[0][0];
+            const id = this.state.market.metrics.get('item').putForSaleAmount;
```

Comment:
"amount" is a formal term in ERTP.

please rename to something about how this is is a serial counter. Also consider making clear that it's to be incremented at the end (that it's the count of items that have already been successfully listed for sale).